### PR TITLE
Different fixes for word usage examples

### PR DESCRIPTION
**Pull request by irina060981**
1. For the https://github.com/alpheios-project/components/issues/610
changed sorting option

2. For the https://github.com/alpheios-project/components/issues/611
changed filter -> limit

### CODE REVIEW

File: src/locales/en-us/messages.json [hunk 1]
```diff
@@ -600,17 +600,17 @@
...
   },
   "WORDUSAGE_FILTERS_AUTHOR_CLEAR": {
-    "message": "Clear author filter",
+    "message": "Clear author limit",
     "description": "Word Usage Examples Type filter",
     "component": "WordUsageExamplesFilters"
   },
   "WORDUSAGE_FILTERS_TEXTWORK_CLEAR": {
-    "message": "Clear textwork filter",
+    "message": "Clear textwork limit",
```
Comment (balmas) [inline src/locales/en-us/messages.json]: please change "textwork" to "work" here too. thanks!

File: src/locales/en-us/messages-word-list.json [hunk 2]
```diff
@@ -95,17 +95,17 @@
         "component": "WordFilterPanel"
     },
     "WORDLIST_FILTER_BY": {
-        "message": "Filter by",
+        "message": "Limit by",
```
Comment (balmas) [inline src/locales/en-us/messages-word-list.json]: I think it should be "Limit to"

File: src/locales/en-us/messages-word-list.json [hunk 3]
```diff
@@ -95,17 +95,17 @@
...
         "description": "Filter by label on the panel",
         "component": "WordFilterPanel"
     },
     "WORDLIST_FILTER_CLEAR": {
-        "message": "Clear filter by",
+        "message": "Clear limit by",
```
Comment (balmas) [inline src/locales/en-us/messages-word-list.json]: can we say "Clear Limits"

File: src/locales/en-us/messages.json (hunk 1 above)
Comment (irina060981) [reply src/locales/en-us/messages.json]: Oh, missed it - fixed

File: src/locales/en-us/messages-word-list.json (hunk 2 above)
Comment (irina060981) [reply src/locales/en-us/messages-word-list.json]: Fixed it - sorry for my poor english!

File: src/locales/en-us/messages-word-list.json (hunk 3 above)
Comment (irina060981) [reply src/locales/en-us/messages-word-list.json]: Fixed

**Comment by irina060981** [comment]
Also I was going to add changes for word usage get result worflow by the following issues
https://github.com/alpheios-project/components/issues/619
https://github.com/alpheios-project/components/issues/620

I have removed Get results and Get results without limits because it doesn't make any sense without Get results button.
I have added auto filter on changing filtering properties.

Hope it looks better!

File: src/locales/en-us/messages-word-list.json (hunk 2 above)
Comment (balmas) [reply src/locales/en-us/messages-word-list.json]: Either is grammatically correct :-)

**Comment by irina060981** [comment]
> looks ok, except I'm not sure about the removal of the collapsed header. We still want the ability to show or hide the filters (don't we?)

I didn't remove the collapsed header? I only put all filtering and sorting staff into it - like this:
First open

image

click show limits

image

select author in filter
image

**Comment by irina060981** [comment]
As everything else is reviewed and has no changes requested - I will go ahead and merge it!

**Pull request merged**